modules/PhyVersoBSP/phyverso_config/evConfig.cpp

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include "evConfig.h"
+#include "phyverso.pb.h"
+#include <fstream>
+#include <iostream>
+
+// for convenience
+using json = nlohmann::json;
+
+evConfig::evConfig() {
+}
+
+evConfig::~evConfig() {
+}
+
+bool evConfig::open_file(std::string path) {
+    try {
+        std::ifstream f(path);
+        config_file = json::parse(f);
+        // check validity first
+        return check_validity();
+    } catch (const std::exception& e) {
+        std::cerr << "error: " << e.what() << "\n";
+    } catch (...) {
+        std::cerr << "Exception of unknown type!\n";
+    }
+    return false;
+}
+
+// todo: needs to be refactored a lot
+bool evConfig::check_validity() {
+    std::cout << config_file.dump() << "\n\n";
+
+    if (!config_file.contains("motor_lock_type")) {
+        printf("Missing 'motor_lock_type' config parameter\n");
```

Comment:
Mixing `std::{cout,cerr}` and `printf()` in the same unit looks inconsistent. (The latter requires `<cstdio>`, if I am not mistaken.)